components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/AbstractAPIManager.java

```diff
@@ -886,6 +886,94 @@ public API getAPI(APIIdentifier identifier,APIIdentifier oldIdentifier, String o
     /**
      * Returns a list of pre-defined # {@link org.wso2.carbon.apimgt.api.model.Tier} in the system.
      *
+     * @return Set<Tier>
+     */
+    public Set<Tier> getAppTiers() throws APIManagementException {
+        Set<Tier> tiers = new TreeSet<Tier>(new TierNameComparator());
+
+        Map<String, Tier> tierMap;
+        if (tenantId == 0) {
+            tierMap = APIUtil.getAppTiers();
+        } else {
+            PrivilegedCarbonContext.startTenantFlow();
+            PrivilegedCarbonContext.getThreadLocalCarbonContext().setTenantId(tenantId, true);
+            tierMap = APIUtil.getAppTiers(tenantId);
+            PrivilegedCarbonContext.endTenantFlow();
```

Comment:
already removed in commit https://github.com/wso2/carbon-apimgt/commit/f7c4cb6913c12e654009faf2a6631ebf2a9cefc4